### PR TITLE
HISAT2 Cluster workaround

### PR DESCRIPTION
**Pull request by fsanchogomez**
FIxes a small `$TMPDIR` formatting mistake in `rules/fastq_map.snakefile` and adds a note to `README.md` explaining a workaround for running HISAT2 in case of problems related to the `$TMPDIR` variable in some computing clusters.

### CODE REVIEW

**Review by vivekbhr — COMMENTED** [review]
@fsanchogomez I left 2 comments. Have a look

File: rules/fastq_map.snakefile
[the remark this answers is not in the document]
Comment (fsanchogomez) [reply rules/fastq_map.snakefile:1]: I was doing a little experiment with how the threading works in snakemake and it slipped through the commit, sorry. I just fixed it.

File: rules/map_scRNAseq.snakefile [hunk 1]
```diff
@@ -84,7 +84,6 @@ rule rna_mapReads:
         mem_mb=100000
     shell:
         """
-        TMPDIR={config[tempDir]}
         MYTEMP=$(mktemp -d ${{TMPDIR:-/tmp}}/snakepipes.XXXXXXXXXX);
```
[the remark this answers is not in the document]
Comment (fsanchogomez) [reply rules/map_scRNAseq.snakefile:87]: Yes, `$TMPDIR` is always populated by the main scChICflow process.
The variable gets recorded on `scChICflow_config.yaml` when it is produced.
In cluster execution, the `$TMPDIR` specified in the `profile/config.yaml` overrides the former if it is different and throws a warning.
I can add an explicit check to the rule if you want if you want me to.

[the remark this answers is not in the document]
Comment (vivekbhr) [reply rules/map_scRNAseq.snakefile:87]: No need to add an explicit check in the rule. As long as the TMPDIR variable is present the code would work.

**Review by vivekbhr — APPROVED** [review]

**Pull request merged**